Content.Server/Power/Nodes/CableNode.cs

```diff
@@ -44,11 +43,11 @@ public override IEnumerable<Node> GetReachableNodes()
                     if (dir == Direction.Invalid)
                     {
                         // On own tile, block direction it faces
-                        terminalDirs |= 1 << (int) IoCManager.Resolve<IEntityManager>().GetComponent<TransformComponent>(node.Owner).LocalRotation.GetCardinalDir();
+                        terminalDirs |= 1 << (int) entMan.GetComponent<TransformComponent>(node.Owner).LocalRotation.GetCardinalDir();
```

Comment:
Should this one also have EntityQuery<TransformComponent> passed in?

Reply:
This is the only `GetReachableNodes()` that uses target xforms, but yeah I guess there is no real reason not to just include it as an argument.

Reply:
now included